doc/1.manual/x2.htm

```diff
@@ -447,9 +447,10 @@ <H3> <A id="s2.7">2.2.7. Context menu for 'Properties', 'Open' and 'Help'</A> </
 
 <P>All the "clicking" mentioned above is done with the left mouse button.
 The right button, instead, opens a context menu offering "Properties",
-"Open" and "Help" (either in edit or run mode). For Apple users who may
-only have one button on their mouse, double-clicking or <kbd>Alt + Click</kbd>
-are mapped to right-click and on trackpads you can click with two fingers.
+"Open" and "Help" (either in edit or run mode). Apple users, who may
+only have one button should know that <kbd>Control + Click</kbd> is mapped
+to right-click and that new magic mousw dwvices can map gestures when you
```

Comment:
you might want to run a spwll chwckwr

Reply:
there's also:
```
x2.htm:105: pathc ==> patch
x2.htm:270: reffered ==> referred
x2.htm:1636: resonsible ==> responsible
x4.htm:549: differeces ==> differences
x5.htm:39: obects ==> objects
x6-c.htm:81: allows to ==> allows one to
```

Reply:
ok, will do

Reply:
I was actually using CODESPELL, what do you suggest? I thought "spwll chwckwr" was a real thing with a funny name but I can't find anything about it, which makes me believe that ironically you didn't check your spelling

Reply:
@porres there is "mousw dwvices" in the quoted text. same error type. :)

Reply:
oh, I see @umlaeute  was making fun of my mistake :) but yewh, I'vw bwwn using codwspwll from timw to timw!

Reply:
what a twrriblw mistakw that was... by thw way